backend/src/test/java/com/google/sps/RemoveParticipantHelperTest.java

```diff
@@ -0,0 +1,61 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.google.sps.datastore.ParticipantDatastore;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mock;
+
+@RunWith(JUnit4.class)
+public final class RemoveParticipantHelperTest {
+
+  @Mock UsernameService usernameService = mock(UsernameService.class);
+  @Mock HttpServletRequest request = mock(HttpServletRequest.class);
+  @Mock HttpServletResponse response = mock(HttpServletResponse.class);
+  @Mock ParticipantDatastore participantDatastore = mock(ParticipantDatastore.class);
```

Comment:
Another consistency comment. I currently have the mock(....class) part in a setUp method since I also use datastore in my tests.

Reply:
Since there is only one functionality being tested in this test class, I don't think there is a need for a setUp method here and it can instead be done in-line within the declaration.

Reply:
Either way works. My only suggestion here would be to remove the @Mock annotation and make these private final fields of the test. The @Mock annotation is meant to work with a call to Mockito.initMocks(testClassInstance) which is usually placed in setUp and is an alternative to manually calling mock as you have here. Without a call to initMocks the @Mock annotation does nothing.

Reply:
Removed the @Mock annotations, thank you for the suggestion!